vcd/resource_vcd_edgegateway.go

```diff
@@ -292,26 +310,29 @@ func setEdgeGatewayValues(d *schema.ResourceData, egw govcd.EdgeGateway) error {
 	if err != nil {
 		return err
 	}
+	var gateways = make(map[string]string)
 	var networks []string
 	for _, net := range egw.EdgeGateway.Configuration.GatewayInterfaces.GatewayInterface {
 		if net.InterfaceType == "uplink" {
 			networks = append(networks, net.Network.Name)
+			gateways[net.SubnetParticipation.Gateway] = net.Network.Name
 		}
 	}
 	err = d.Set("external_networks", networks)
 	if err != nil {
 		return err
 	}
 
-	err = d.Set("advanced", egw.EdgeGateway.Configuration.AdvancedNetworkingEnabled)
-	if err != nil {
-		return err
-	}
-	err = d.Set("ha_enabled", egw.EdgeGateway.Configuration.HaEnabled)
-	if err != nil {
-		return err
-	}
+	_ = d.Set("advanced", egw.EdgeGateway.Configuration.AdvancedNetworkingEnabled)
+	_ = d.Set("ha_enabled", egw.EdgeGateway.Configuration.HaEnabled)
+	_ = d.Set("distributed_routing", egw.EdgeGateway.Configuration.DistributedRoutingEnabled)
```

Comment:
This conflicts with the "TODO" comment couple of lines below. Was there a bug/mistake in the previous assumption?

Reply:
My bad. I got carried away and did not check my own notes. Removing.